sticky.js

```diff
@@ -88,7 +88,7 @@ var Sticky = React.createClass({
 
 
   isModernBrowser: function() {
-    return requestAnimationFrame && cancelAnimationFrame;
+    return window.requestAnimationFrame && window.cancelAnimationFrame;
```

Comment:
Would you maybe want to do something like `return window && window.requestAnimationFrame && window.cancelAnimationFrame` to guard against usage in any non-browser environments where window may be undefined, or is that overkill?